### PR TITLE
fix: faq hash problem

### PR DESCRIPTION
**Pull request by Benehiko**
## Related issue

https://github.com/ory/kratos/pull/1105

## Proposed changes

Fix the production build css class hashing problem. This will make sure the production build does not break.

## Checklist

- [X] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [X] I have read the [security policy](../security/policy).
- [X] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [X] I have added or changed [the documentation](docs/docs).

## Further comments

@zepatrik

### CODE REVIEW

File: docs/config.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 module.exports = {
   projectName: 'ORY Kratos',
   projectSlug: 'kratos',
+  baseUrl: '/',
```
Comment (zepatrik) [inline docs/config.js:4]: Revert that

File: docs/docs/faq.mdx [hunk 2]
```diff
@@ -1,179 +0,0 @@
----
```
Comment (zepatrik) [inline docs/docs/faq.mdx:1]: For some reason the new file is not included, generate and commit it :wink:

File: docs/scripts/gen-faq.js [hunk 3]
```diff
@@ -41,9 +41,10 @@ import {Question, Faq} from '@theme/Faq'
 md = new Remarkable()
 faq.forEach((el) => {
   react_tags = el.tags.map((tag) => {
-    return tag + '_src-theme-'
+    return tag
```
Comment (zepatrik) [inline docs/scripts/gen-faq.js:44]: The whole `.map()` here is redundant as it does nothing

File: docs/src/theme/Faq.js [hunk 4]
```diff
@@ -1,19 +1,22 @@
...
     className={cn(
-      { [styles.selected]: isSelected },
-      tag + '_src-theme-',
-      'pills',
-      'pills__item',
-      { 'styles.pills__item--active': isSelected }
+      { [genStyle.selected]: isSelected },
+      genStyle[tag],
+      styles.pills,
+      styles.pills__item,
```
Comment (zepatrik) [inline docs/src/theme/Faq.js:18]: @k9ert what are these classes doing in the first place? We figured out that they where not applied before, and nothing changed when we did apply them :sweat_smile:

**Review by zepatrik — CHANGES_REQUESTED** [review]
We will have to move the changes into https://github.com/ory/docusaurus-template/tree/master/src/theme anyway, but this is a POC that the fix is working.

File: docs/scripts/gen-faq.js (hunk 3 above)
Comment (Benehiko) [reply docs/scripts/gen-faq.js:44]: wuut, i removed those.

Comment (Benehiko) [reply docs/scripts/gen-faq.js:44]: ahh yes you're right

File: docs/docs/faq.mdx (hunk 2 above)
Comment (Benehiko) [reply docs/docs/faq.mdx]: uhh, lemme fix that quick

**Review by Benehiko — COMMENTED** [review]
updated with the necessary changes

**Comment by zepatrik** [comment]
Ok nice, I will port that to docusaurus-template now :+1:

File: docs/src/theme/Faq.js (hunk 4 above)
Comment (zepatrik) [reply docs/src/theme/Faq.js:18]: Nevermind, it does look better now.

**Comment by zepatrik** [comment]
Closing as this is fixed now :tada:

**Pull request closed without merging**